### PR TITLE
Add a CONTRIBUTING file prior to turning on DCO enforcement

### PR DESCRIPTION
**Pull request by rminnich**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+We have a few rules not covered by these tools:
+
+- Standard imports are separated from other imports. Example:
+    ```
+    import (
+      "regexp"
+      "time"
+
+      dhcp "github.com/krolaw/dhcp4"
```
Comment (insomniacslk) [inline CONTRIBUTING.md]: this is going to change soon! :P

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    )
+    ```
+
+## Patch Format
+
+Well formatted patches aide code review pre-merge and code archaeology in
```
Comment (insomniacslk) [inline CONTRIBUTING.md]: aide? Aid? Or TIL a new word for me?

Comment (rminnich) [reply CONTRIBUTING.md]: Fixed (aid)

File: CONTRIBUTING.md (hunk 1 above)
Comment (rminnich) [reply CONTRIBUTING.md]: it's not important here, it's just an example.

Comment (insomniacslk) [reply CONTRIBUTING.md]: I know, I was just joking

Comment (JulienVdG) [reply CONTRIBUTING.md]: If I'm not mistaken this is what goimports does as a formatter instead of gofmt right ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+## General Guidelines
+
+## Pull Requests
+
+We accept GitHub pull requests.
+
+Fork the project on GitHub, work in your fork and in branches, push
+these to your GitHub fork, and when ready, do a GitHub pull requests
+against https://github.com/u-root/u-root.
```
Comment (JulienVdG) [inline CONTRIBUTING.md]: Why u-root and not fiano here ?

Comment (rjoleary) [reply CONTRIBUTING.md]: Fixed in #258

**Pull request merged**